requirements.txt

```diff
@@ -1,2 +1,2 @@
-torch>=1.6
```

Comment:
is there any reason to freeze again PT version?

Reply:
We could have it up and above. 
FYI. This PR is also trying to fix the MNIST issue. The torchvision team has pushed the User-Agent fix in the new release.

Reply:
@Borda

Reply:
understand the motivation but ban all lover `torchvision` version only because of downloading dataset?
rather just implement the fix, see https://github.com/PyTorchLightning/pytorch-lightning/pull/6357/files#r588936381

Reply:
Yup, I get you. There are a decent number of tests depending on the Dataset. Let me push the fix.

Reply:
WIP. Will get to this tomorrow morning.